apps/stack/template.yaml

```diff
@@ -42,6 +42,12 @@ Metadata:
           - DatasourceID
           - GQLToken
           - UpdateTimestamp
+      - Label:
+          default: Enable Observe Poller
```

Comment:
I know Joao had you switch it back to this but we're going to add another Label + parameter group here later for config poller right? And that'll be separate? It actually seems like we would want to differentiate between the two pollers vs just calling this Observe Poller

Reply:
My idea was that we could just integrate the roles necessary for both pollers into one role/policy. I agree that for now the best term is `Metrics Poller`. When the next poller comes in, I thought that we could choose to either separate it out/call it `Enable Observe Pollers`

Reply:
got it, so actually it won't be a separate label/parameter group for config pollers?

Reply:
Honestly I haven't scoped out the amount of new roles that would be required, but I suspect it would be pretty limited, like some AllowReadResources or something like that, my thought is that's not enough to justify a whole other section.